client/src/SessionsList.js

```diff
@@ -29,16 +29,19 @@ const useStyles = makeStyles((theme) => ({
     },
     '& .MuiTypography-root': {
       color: 'white',
-    },
-  },
-  fab: {
-    position: 'sticky',
+    },EditOutlined
```

Comment:
```suggestion
    },
  },
  fab: {
    position: 'sticky',
```

something went terribly wrong here :)

Reply:
I suggest installing an eslint plugin or running `yarn lint` inside `/client` directory - it will catch errors like this easily :)